src/main/java/org/opensearch/ad/caching/CacheBuffer.java

```diff
@@ -449,11 +460,19 @@ public String getDetectorId() {
         return detectorId;
     }
 
-    public List<ModelState<?>> getAllModels() {
+    public List<ModelState<EntityModel>> getAllModels() {
         return items.values().stream().collect(Collectors.toList());
     }
 
     public PriorityTracker getPriorityTracker() {
         return priorityTracker;
     }
+
+    public void setMinimumCapacity(int minimumCapacity) {
```

Comment:
Seems the `minimumCapacity` means minimum entity count? Add some comments or change the variable name ?

Reply:
It means the reserved cache size.  So no matter how many entities there are, we will keep the size for minimum capacity entities.  Added the comments.